ferry.go

```diff
@@ -1021,3 +1030,88 @@ func (f *Ferry) checkConnectionForBinlogFormat(db *sql.DB) error {
 
 	return nil
 }
+
+func (f *Ferry) UpdateBatchSizes() error {
```

Comment:
IMO it's a bit weird to have this method, which acts on the Config to be a function attached to the Ferry. I understand that you need access to the source db connection, but maybe it's better/less surprising to have this method be attached to the config (or even just the DataIterationBatchSizePerTableOverride struct) and take a source db connection?

Reply:
I agree, it makes more sense to be on the `DataIterationBatchSizePerTableOverride` struct